### ISSUE TITLE
Model Quiz 4

### ISSUE THREAD

**Issue by jjyock** [jjyock/Apply-machine-learning-in-Python#2]
1.Which of the following is an example of clustering?
2.Which of the following are advantages to using decision trees over other models? (Select all that apply)
Trees often require less preprocessing of data
Trees are easy to interpret and visualize
3.What is the main reason that each tree of a random forest only looks at a random subset of the features when building each node?
To improve generalization by reducing correlation among the trees and making the model more robust to bias.
4.Which of the following supervised machine learning methods are greatly affected by feature scaling? (Select all that apply)
Support Vector Machines KNN
5.Select which of the following statements are true.
For a model that won’t overfit a training set, Naive Bayes would be a better choice than a decision tree.
For predicting future sales of a clothing line, Linear regression would be a better choice than a decision tree regressor.
6.
7.
9.Which of the following might be good ways to help prevent a data leakage situation?
If time is a factor, remove any data related to the event of interest that doesn’t take place prior to the event.
Remove variables that a model in production wouldn’t have access to, 
Sanity check the model with an unseen validation set
10. 0 1 1 0
